cvat/apps/dashboard/templates/dashboard/dashboard.html

```diff
@@ -143,6 +143,33 @@
                             <input type="checkbox" id="dashboardCustomQuality" title="Custom image quality"/>
                         </td>
                     </tr>
+                    <tr>
+                        <td>
+                            <label class="regular h2"> Start Frame </label>
+                        </td>
+                        <td>
+                            <input type="number" id="dashboardStartFrame" class="regular" style="width: 4.5em;" max="2147483647" min="0" value=0 disabled=true/>
+                            <input type="checkbox" id="dashboardCustomStart" title="Custom start frame"/>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <label class="regular h2"> Stop Frame </label>
+                        </td>
+                        <td>
+                            <input type="number" id="dashboardStopFrame" class="regular" style="width: 4.5em;" max="2147483647" min="0" value=0 disabled=true/>
```

Comment:
What is the reason to have max value for start fame, stop frame and step?

Reply:
The max value just a placeholder. I can remove it.

Reply:
Please remove.